packages/test-utils/src/api-client.ts

```diff
@@ -184,26 +187,29 @@ export class ApiClient {
   }
 
   /**
-   * Helper method to handle API errors consistently
+   * Helper method to handle API errors consistently.
+   * Logs at debug level since many tests intentionally trigger errors.
    */
   private handleApiError(error: unknown, operation: string): ErrorResponse {
-    console.error(`Failed to ${operation}:`, error);
```

Comment:
actually..._this_ was contributing to a ton of noise. 

instead, we remove it and only log test API issues when in debug mode.